src/components/molecules/AddToDashboard.tsx

```diff
@@ -21,11 +22,13 @@ const AddToDashboard: React.FC<Props> = (props) => {
 
     const add = () => {
         dispatch(dashboard.actions.add(props.dashboardItem));
+        toaster.success('Visualiseringen ble lagt til i dashboardet.');
         history.push('/');
     };
 
     const remove = () => {
         dispatch(dashboard.actions.remove(key));
+        toaster.success('Visualiseringen ble fjernet fra dashboardet.');
```

Comment:
Is deleting a `success` action? I think it may look more nuanced with `info`.

Please share your thoughts.

Reply:
I think success is the most fitting for both adding and removing, since it conveys to the user that it succeeded in doing what the user asked. Having consistent feedback for these operations (adding/deleting) is good, since these are the operations in which toasts are used. _These are (naturally) my subjective thoughts._

How _I_ think the logic of toasters should be:
* `success`: any user operation that was successful (e.g. "Successfully deleted the entire database")
* `notify`/`info`: useful information not necessarily related to user interacting with the page (e.g. "Check out this offer")
* `warning`: trivial user mistakes such as not filling in a form (e.g. "Please fill in your email") 
* `error`:  errors rooted in a user action that failed (not necessarily caused by the user, could be a server error: "Could not update username: `${error.message}`")

Let me know if this is totally stupid 🚀

Reply:
Good reflections! I usually use notify and info for more user-interactive actions, but consistency is the most important.